backend/papersfeed/utils/recommendations/utils.py

```diff
@@ -86,7 +86,7 @@ def select_recommendation(args):
 
     recommendation_queryset = UserRecommendation.objects.filter(user_id=request_user.id).order_by('rank')
 
-    recommendations = get_results_from_queryset(recommendation_queryset, 20, page_number)
+    recommendations = get_results_from_queryset(recommendation_queryset, 30, page_number)
```

Comment:
I think 30 does not work because `get_results_from_queryset()` has the part `pages = Paginator(queryset, min(20, count))`. What was your intention and did it work as you wanted?

Reply:
Oh, I forgot about that. I wanted to pass 30 recommendations at the first when there is no subscription for the current user, because there would be 30 feeds(20 subs and 10 recos) at the normal case.

Reply:
@kosumin So will you change codes for it?

Reply:
Yes, I think I just have to change the code related to backend pagination from 20 to 30.